src/main/kotlin/com/parmet/buf/gradle/BufExtension.kt

```diff
@@ -87,5 +87,10 @@ class GenerateOptions(
      * If you specify any dependencies in `buf.yaml`, you must create a `buf.lock` file using `buf mod update` for
      * dependency resolution to succeed.
      */
-    var includeImports: Boolean? = null
+    var includeImports: Boolean? = null,
+
+    /**
+     * Specify the location of buf.gen.yaml.
+     */
+    var genFileLocation: File? = null
```

Comment:
The Buf docs refer to this as a template file. Should this property be called `templateFileLocation`?

Reply:
This makes sense to me 👍